<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageTabsDialog</class>
 <widget class="QDialog" name="ManageTabsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TAC Configuration Editor - Manage Tabs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="_tabsTable">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visible</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User Defined</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Configurable</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Moveable</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="_addTabButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/TACConfigEditor.qrc">
         <normaloff>:/TabsPlus.png</normaloff>:/TabsPlus.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_removeTabButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/TACConfigEditor.qrc">
         <normaloff>:/TabsRemove.png</normaloff>:/TabsRemove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_renameTabButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/TACConfigEditor.qrc">
         <normaloff>:/TabsRename.png</normaloff>:/TabsRename.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>29</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="_moveTabUpButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources/TACConfigEditor.qrc">
           <normaloff>:/TabsUp.png</normaloff>:/TabsUp.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_moveTabDownButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources/TACConfigEditor.qrc">
           <normaloff>:/TabsDown.png</normaloff>:/TabsDown.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/TACConfigEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManageTabsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManageTabsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
